Modules/Loadable/SubjectHierarchy/Widgets/Python/SubjectHierarchyPlugins/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+
+# Deploy abstract scripted plugin and init file that
+# imports all deployed scripted plugins deployed by modules
+set(SubjectHierarchyPlugins_PYTHON_SCRIPTS
+  __init__
+  AbstractScriptedSubjectHierarchyPlugin
+  parameterNodeWrapper/__init__
```

Comment:
`Modules\Loadable\SubjectHierarchy\Widgets\Python\SubjectHierarchyPlugins\parameterNodeWrapper` folder is misplaced: `parameterNodeWrapper` has nothing to do with `SubjectHierarchyPlugins`. It is a ParameterNodeWrapper plugin, therefore it must be in `Modules\Loadable\SubjectHierarchy\Widgets\Python\ParameterNodeWrapperPlugins` folder. Note that the folder name was not following Python package naming conventions either (starting with lowercase), the proposed name fixes that, too.

Reply:
Since the `parameterNodeWrapper` package is namespaced within `SubjectHierarchyPlugins`, i don't think it is an issue. This is the intended approach.

Reply:
I suggest we address the `parameterNodeWrapper` in a follow up commit. This PR fixes the regression without impacting the `parameterNodeWrapper`

Reply:
It is true that it may work as is, but it is not the right thing to do.

SubjectHierarchyPlugins folder must not contain ParameterNodeWrapper Plugins. Probably it was misleading that the SubjectHierarchyPlugins folder name could be interpreted as "a folder that contains all Python plugins for the SubjectHierarchy module". But in fact this is "a shared folder for storing SubjectHierarchyPlugins in any module". To be consistent with how all the other Python plugins are defined in Slicer, ParameterNodeWrapperPlugins folder should contain ParameterNodeWrapper plugins of all Slicer core modules.